apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-bot-routing.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { createFakeVisitor } from '../../mocks/data';
+import { IS_EE } from '../config/constants';
+import { createAuxContext } from '../fixtures/createAuxContext';
+import { Users } from '../fixtures/userStates';
+import { HomeOmnichannel, OmnichannelLiveChat } from '../page-objects';
+import { test, expect } from '../utils/test';
+
+test.describe('OC - Bot Agent Routing Enabled', () => {
+	test.skip(!IS_EE, 'Enterprise Only');
+
+	// User1 will be the bot agent
+	let poHomeOmnichannel: HomeOmnichannel;
+
+	// User2 will be the human agent
+	let poHomeOmnichannelUser2: HomeOmnichannel;
+
+	let poLiveChat: OmnichannelLiveChat;
+
+	const visitorBot = createFakeVisitor();
+	const visitorHuman = createFakeVisitor();
```

Comment:
I think this should be located inside some before, otherwise we can have issues during retries